ihatemoney/models.py

```diff
@@ -164,6 +165,32 @@ def monthly_stats(self):
     def uses_weights(self):
         return len([i for i in self.members if i.weight != 1]) > 0
 
+    def get_monthly_expenditure(self, member_id):
```

Comment:
As a non-native english-speaker, I'm not familiar with the term "expenditures". How does it differs with "expenses"? If there is no difference, should we keep using "expenses" instead?

Reply:
I am also a non-native English speaker and I just looked it up and I think although they are similar, "expenses" is probably more appropriate too. I'll change it!